docs/hellonode.md

```diff
@@ -227,30 +270,82 @@ docker push gcr.io/PROJECT_ID/hello-node:v2
 
 Building and pushing this updated image should be much quicker as we take full advantage of the Docker cache.
 
-We’re now ready for kubernetes to smoothly update our replication controller to the new version of the application:
+We’re now ready for kubernetes to smoothly update our deployment to the new version of the application.  In order to change
+the image label for our running container, we will need to edit the existing *hello-node deployment* and change the image from
```

Comment:
"Find the image field under `.spec.template.spec.containers` and change it from..."